AzureMonitorAgent/ama_tst/modules/main.py

```diff
@@ -60,7 +66,62 @@ def run_troubleshooter():
             collect_logs()
             return
 
+        # silent vs interactive mode
+        print("--------------------------------------------------------------------------------")
+        print("The troubleshooter can be run in two different modes.\n"\
+            "  - Silent Mode runs through with no input required\n"\
+            "  - Interactive Mode includes extra checks that require input")
+        mode = get_input("Do you want to run the troubleshooter in silent (s) or interactive (i) mode?",\
+                        (lambda x : x.lower() in ['s','silent','i','interactive','q','quit']),\
+                        "Please enter 's'/'silent' to run silent mode, 'i'/'interactive' to run \n"\
+                            "interactive mode, or 'q'/'quit' to quit.")
+        if (mode.lower() in ['q','quit']):
+            print("Exiting the troubleshooter...")
+            return
+        elif (mode.lower() in ['s','silent']):
+            print("Running troubleshooter in silent mode...")
```

Comment:
is there a way that the troubleshooter can be entirely silent mode -- that is, a single command like `./entrypoint.py --some-option --another-option` and the TST will run to completion? my reading of it currently is you still need to choose the run mode _after_ starting TST (rather than being able to specify runmode in the initial TST start command).

Reply:
Yes the current design is to choose silent/interactive mode after starting it. This follows the design of OMS troubleshooter, but it seems good to have a command line option to start it as a silent mode without extra input

Reply:
Can I add flag option in a separate PR?

Reply:
Sure, no need to block this PR on this. Can be a future enhancement